tests/TCP_echo_server.py

```diff
@@ -165,7 +165,11 @@ def run(self):
                     sck = socket.socket(socket.AF_INET, socket.SOCK_STREAM, 0)
                     sck.bind((self.HOST, int(self.port)))
                     self.sock = context.wrap_socket(sck, server_side=True)
-                    self.sock.listen(5)
+                    # self.sock.listen(n) - where n is the number of unaccepted connections that the
+                    # system will allow before refusing new connections.
+                    # Since the CI system is low on resources, it might take time for the echo server
+                    # to accept TLS connections. Test setting the value of n to 25
+                    self.sock.listen(25)
```

Comment:
Alternatively, you can set it to socket.SOMAXCONN, which is the system default maximum.

Reply:
Done ! Thanks.